src/sentry/static/sentry/app/components/projectSelector.jsx

```diff
@@ -34,6 +35,9 @@ class ProjectSelector extends React.Component {
     multiProjects: PropTypes.arrayOf(
       PropTypes.oneOfType([PropTypes.string, SentryTypes.Project])
     ),
+    nonMemberProjects: PropTypes.arrayOf(
+      PropTypes.oneOfType([PropTypes.string, SentryTypes.Project])
```

Comment:
When would this prop be a string?

Reply:
Ah, it won't be. Updated to only accept Project